### PR TITLE
OADP 1.2.0 release notes

### PR DESCRIPTION
**Pull request by HagayVider1**
OCPBUGS# : [OADP-1181](https://issues.redhat.com//browse/OADP-1181)
OSDDOCS# : [OADP-1181](https://issues.redhat.com//browse/OADP-1181)

OADP 1.2.0 release notes

Version(s): OADP 1.2.0

Issue: https://issues.redhat.com/browse/OADP-1181

Link to docs preview:

QE review: https://61156--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-release-notes.html
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by HagayVider1** [comment]
@weshayutin  @akarol @kaovilai 
This PR replaces the previous PR - https://github.com/openshift/openshift-docs/pull/60969

**Comment by HagayVider1** [comment]
@weshayutin @PrasadJoshi12 - fixed build issues
Please review "Technical Preview" section.
If you don't want it in the release notes, I can comment it out.

**Comment by anarnold97** [comment]
@HagayVider1 - I think we could probably lose these from the Known Issues:

OADP-1976 - QE automation test
OADP-1358 - feature request
OADP-1231 - Sub-task?
OADP-1062 - downstream effort

And possibly even - 

OADP-966 - task
OADP-987 - task
OADP-1354 - task

**Please check before removing though**

**Comment by anarnold97** [comment]
@HagayVider1 the format of the known issues is wrong:
image

**Comment by anarnold97** [comment]
Ok, so the only definite known is [OADP-1933](https://issues.redhat.com/browse/OADP-1933)

Datamover backup over CephFS failed with "Asynchronous action timed out"

There is a link on the ticket that will allow you to do the description.

@weshayutin - please can you confirm this is the only known issue that you want to highlight?

**Comment by HagayVider1** [comment]
@anarnold97 I also found OADP-1861 and OADP-1136 are QE tickets. I removed them.
@weshayutin @PrasadJoshi12  - Changed "Known Issues" to bulleted list with descriptions.

**Comment by HagayVider1** [comment]
@anarnold97 @weshayutin @PrasadJoshi12  - Removed OADP-661 from Known Issues

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:81]: @HagayVider1 - you cannot write a known issues section for release notes without reading the ticket.
There is a nice little line in it, that gives a useable section: 

OADP 1.2.x no longer requires VolSync's privledged movers and can cause errors. A user could see a failed backup that timed out with the following errors:

```
Error: container's runAsUser breaks non-root policy 
Error from server (BadRequest): container "restic" in pod "volsync-src-vsb-xxxyyy" is waiting to start: CreateContainerConfigError
```

Comment (anarnold97) [reply modules/oadp-release-notes-1-2-0.adoc:81]: Then we could add something else reworked from the Q&A:

By default, OADP uses the Pod/Workloads security context to backup data using Volsync/datamover.  Also by default, `dpa.spec.features.dataMover.volumeOptions.sourceVolumeOptions.moverSecurityContext` is set to `true`.

Issue the following command to ensure the annotation is set to `false`:
```
oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'
```

> [!NOTE]
>
>  In some more advanced use case where you may want and need root escalation of privileges the option to use the annotation and setting `moverSecurityContext` to false.

Comment (amastbau) [reply modules/oadp-release-notes-1-2-0.adoc:81]: afaiu/r  moverSecurityContext defaults to false,
maybe double-check?

Comment (amastbau) [reply modules/oadp-release-notes-1-2-0.adoc:81]: https://issues.redhat.com/browse/OADP-1941?jql=text%20~%20%22moverSecurityContext%22

**Review by amastbau — CHANGES_REQUESTED** [review]

File: modules/oadp-release-notes-1-2-0.adoc
Comment (HagayVider1) [reply modules/oadp-release-notes-1-2-0.adoc]: > Then we could add something else reworked from the Q&A:
> 
> By default, OADP uses the Pod/Workloads security context to backup data using Volsync/datamover. Also by default, `dpa.spec.features.dataMover.volumeOptions.sourceVolumeOptions.moverSecurityContext` is set to `true`.
> 
> Issue the following command to ensure the annotation is set to `false`:
> 
> ```
> oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'
> ```
> 
> > [!NOTE]
> > In some more advanced use case where you may want and need root escalation of privileges the option to use the annotation and setting `moverSecurityContext` to false.

Should I add this to Known issue OADP-1933?

**Comment by HagayVider1** [comment]
@weshayutin @akarol @PrasadJoshi12 @amastbau @anarnold97 
See updated OADP-1933 known issue.

**Comment by anarnold97** [comment]
@HagayVider1 - i still think the Fixed Bugs list is too long. 

I might remove the following as well:

https://issues.redhat.com/browse/OADP-1741 - Feature request

https://issues.redhat.com/browse/OADP-1067 - Task

https://issues.redhat.com/browse/OADP-1332 - Task

https://issues.redhat.com/browse/OADP-1164 - Task

https://issues.redhat.com/browse/OADP-1105 - Epic

https://issues.redhat.com/browse/OADP-969 - Task

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:25]: @HagayVider1 - do you think we should have links to the documentation of the new feature? what do you think?

File: modules/oadp-release-notes-1-2-0.adoc
Comment (weshayutin) [reply modules/oadp-release-notes-1-2-0.adoc]: Folks @anarnold97 for 1.2.0 `oc annotate --overwrite namespace/openshift-adp volsync.backube/privileged-movers='false'`   is no longer required.

**Comment by anarnold97** [comment]
@HagayVider1 - please can you tidy the PR by marking comments you have covered as resolved. I know i usually do it but i cannot do it in this repo

File: modules/oadp-release-notes-1-2-0.adoc
Comment (HagayVider1) [reply modules/oadp-release-notes-1-2-0.adoc:25]: I think we should include links for every new feature. Can you please point me in the direction where these features are documented? Tickets, PRs?

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:17]: Not sure this is a new feature.

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:15]: **availability** -->> **availability**

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:88]: Lose this line? What do you think?

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:84]: **privledged** -->> **privileged**

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [reply modules/oadp-release-notes-1-2-0.adoc:17]: Let us leave it in, I think... but it is your call

File: modules/oadp-release-notes-1-2-0.adoc
Comment (gaurav-nelson) [reply modules/oadp-release-notes-1-2-0.adoc:9]: I think we can simplify it to say:
```
The OADP 1.2.0 release notes include information about new features, bug fixes, and known issues.
```

File: modules/oadp-release-notes-1-2-0.adoc
Comment (gaurav-nelson) [inline modules/oadp-release-notes-1-2-0.adoc:15]:
```suggestion
The new `resourceTimeout` option specifies how many minutes to wait for several Velero resources before timeout. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. Ten minutes is the default setting.
```

- use words if a number occurs at the beginning of a sentence. IBM Styleguide
- `This` was unclear. Use `This option applies ...` or `It applies ...`

File: modules/oadp-release-notes-1-2-0.adoc
Comment (gaurav-nelson) [inline modules/oadp-release-notes-1-2-0.adoc:24]:
```suggestion
The OADP Data Mover enables you to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications using CSI volume snapshots pulled from the object store if a cluster fails, accidental deletion, or corruption occurs.
```

File: modules/oadp-release-notes-1-2-0.adoc
Comment (gaurav-nelson) [inline modules/oadp-release-notes-1-2-0.adoc:88]: missing `,`
```suggestion
By default, OADP uses POD/Workloads security context to backup data using Volsync/Data Mover. Also, by default, `dpa.spec.features.dataMover.volumeOptions.sourceVolumeOptions.moverSecurityContext` is set to `true`.
```

**Comment by gaurav-nelson** [comment]
Thank you for the PR @HagayVider1 🤠  

@anarnold97 I've nitpicked some items and added a few more comments. The PR should be good to merge after those changes.

File: modules/oadp-release-notes-1-2-0.adoc
Comment (snarayan-redhat) [reply modules/oadp-release-notes-1-2-0.adoc:15]: Just one additional nit here. The first sentence sounds a bit incomplete to me. My suggestion (including Gaurav's suggestion above):

```suggestion
The new `resourceTimeout` option specifies the timeout duration in minutes for waiting on various Velero resources. This option applies to resources such as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availability. The default duration is ten minutes.
```

**Review by snarayan-redhat — COMMENTED** [review]
Just one additional nit from me.

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [reply modules/oadp-release-notes-1-2-0.adoc:9]: Thanks Gaurav

File: modules/oadp-release-notes-1-2-0.adoc
Comment (anarnold97) [inline modules/oadp-release-notes-1-2-0.adoc:82]: @weshayutin - please can I confirm if it is ok to remove, as according to QE: "In 'Stage' version the datamover configuration for CephFS was changed. 'Stage' move the configuration  from the 'DPA' to 'openshift-adp namespace' So no need to do anything - namespace configuration done by OADP deployment"

**Comment by HagayVider1** [comment]
DO NOT TOUCH!

**Pull request closed without merging**
